widget/embedded/src/utils/swap.ts

```diff
@@ -487,7 +479,7 @@ export function createQuoteRequestBody(params: {
   affiliateRef: string | null;
   affiliatePercent: number | null;
   affiliateWallets: { [key: string]: string } | null;
-  initialQuote?: BestRouteResponse;
+  initialQuote?: MultiRouteSimulationResult;
```

Comment:
In the earlier version, prior to multi-routing, we utilized the same endpoint for both obtaining a quote and confirming a swap. With the introduction of multi-routing, we can eliminate the checkPrerequisites (types are not updated at the moment; you can add a ts-ignore to remove this property) and blockchains from the request payload since they will not be used in confirming the swap.

Reply:
fixed